lua/nvim_lsp/rust_analyzer.lua

```diff
@@ -1,11 +1,88 @@
 local configs = require 'nvim_lsp/configs'
 local util = require 'nvim_lsp/util'
+local server_name = 'rust_analyzer'
+local release_url = 'https://api.github.com/repos/rust-analyzer/rust-analyzer/releases/latest'
+local postfix = '-linux'
+local bin = 'rust-analyzer'
+
+if vim.fn.has('mac') == 1 then
+  postfix = '-mac'
+elseif vim.fn.has('win32') == 1  or vim.fn.has('win64') == 1 then
+  postfix = '-windows.exe'
+end
```

Comment:
You can omit the comparison with `1`, it's redundant in Lua.

Reply:
redundant how? Only `false` and `nil` are falsy in Lua. `0` is truthy, that's why we compare to `1` since `has()` returns 0 or 1.

Reply:
You're absolutely right. My bad! Ignore this advice.